src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -446,6 +450,12 @@ public int compare(TypedVar o1, TypedVar o2) {
     return treeWalker.valueSymbolsWalked;
   }
 
+  private boolean isValidJSProperty(String other) {
```

Comment:
duh, already spoiled by "extract method", it kept the name of the local var. done